package ult.nodo.ciencia.excepcion;

public class ArticuloException extends GeneralException {

    public ArticuloException(Throwable cause) {
        super(cause);
    }
}
